apps/codeforafrica/src/components/Input/Input.js

```diff
@@ -0,0 +1,69 @@
+import { styled } from "@mui/material/styles";
+import TextField from "@mui/material/TextField";
+import React from "react";
+
+const InputRoot = styled(TextField)(({ theme }) => ({
+  "& .MuiInputLabel-root": {
+    color: theme.palette.text.primary,
+    marginBottom: theme.spacing(0.25),
+    position: "static",
+    transform: "none",
+  },
+  "& .MuiInputLabel-root.Mui-focused": {
+    color: theme.palette.text.primary,
+  },
+  "& .MuiInputLabel-root.Mui-disabled": {
+    color: "#5D5353",
+  },
+  "& .MuiOutlinedInput-root": {
+    borderRadius: 0,
+    backgroundColor: theme.palette.common.white,
+    "&:active fieldset": {
+      borderColor: theme.palette.highlight.main,
+    },
+    "&:hover fieldset": {
+      borderColor: theme.palette.primary.main,
+    },
+    "&.Mui-disabled": {
+      color: "#D0CBCB",
+      backgroundColor: theme.palette.background.main,
+    },
+    "&.Mui-disabled fieldset": {
+      borderColor: theme.palette.grey.light,
+    },
+    "&.Mui-disabled:hover fieldset": {
+      borderColor: theme.palette.grey.light,
+    },
+    "&.Mui-focused fieldset": {
+      borderWidth: 1,
+      borderColor: theme.palette.primary.main,
+    },
+    "&.Mui-error fieldset": {
+      borderColor: theme.palette.error.main,
+    },
+    "& fieldset": {
+      borderColor: "#D0CBCB",
+      top: 0,
+    },
+    "& fieldset legend": {
+      display: "none",
+    },
+  },
+}));
+
+const Input = React.forwardRef(function Input(props, ref) {
```

Comment:
Just out of curiosity, I see the need for `forwardRef` on this Input but I've also seen it in almost all other components that aren't really Inputs; Why is this?

Reply:
It's not strictly required @esirK  but recommended if you're developing reusable components. More on [info](https://reactjs.org/docs/forwarding-refs.html)